contributions/contribution.yaml

```diff
@@ -483,6 +483,34 @@ paths:
           description: Unauthorized
         500:
           description: Internal error
+  /contributions/version:
+    get:
+      summary: Get version of contribution building block
+      tags:
+        - Contributions
+      responses:
+        200:
+          description: Contributions building block version.
+          content:
+            text/plain:
+              schema:
+                type: string
+        400:
+          description: Bad request
+        500:
+          description: Internal server error
+  /contributions/building-blocks:
```

Comment:
Please also include APIKeyAuth to the building blocks endpoint. Thanks.

Reply:
added to both building blocks